### ISSUE TITLE
Unable to use my personal app made for Mac OS on friend's MacBook

### ISSUE THREAD

**Issue by athosflo** [beeware/briefcase#577]
I have created an app with briefcase create, briefcase build, briefcase run and then create a .dmg file.
I shared my app (which is working well on my MacBook) with 3 friends. None of them manage to start the app. There is the usual warning regarding Not Identified developer / Not signed App.
Even if they go to Security and click on Open Anyway, it does not start.
Unfortunately the Beeware tutorial about signing app is empty. Therefore I am stuck with my app, only allowed to work on my MacBook. Can you please share the procedure ?

Thanks for your help

**Comment by freakboy3742** [comment]
Thanks for the report - we definitely need to improve our documentation on app signing for macOS; it's made worse by the fact that Apple has been strengthening those signing requirements with each release of macOS. 

The short version is that when you run `briefcase package`, you can provide a signing identity. You get a signing identity from Apple by joining their developer program; Xcode has tools to synchronise this developer identity.

**Comment by athosflo** [comment]
thanks for that.

Again thank you very much for the beeware project and documentation so far. It is so easy to create an app even for beginners. 3 months ago, I knew nothing about programming.

Could you please give me a quick example on how to identify an app using briefcase package ?
I have signed in on Apple Developer program. But how to add it to briefcase ?

Once it is done, I could write a little tutorial on my Github post to answer my own question and help other people

Thanks again for your help

**Comment by athosflo** [comment]
Hi Russell,
you can disregard previous email. There was just an issue with Xcode and my certificate. 
briefcase package works well now, signing my app. I can now run it on other people’s macbooks. However they have to go to Security settings and click on Open Anyway the first time.

I tried to notorize my app with Xcode using briefcase build macos xcode. 
However when I try to create an archive, I get all these errors

Any idea how to solve this ? 

Best regards
Florian

**Comment by freakboy3742** [comment]
I haven't looked into the notarisation process in detail, so unfortunately, I can't comment. However, I'd be interested in hearing what errors you're seeing; there *might* be enough detail to point you in the right direction.

**Comment by athosflo** [comment]
﻿Here is a screenshot of the errors

**Comment by freakboy3742** [comment]
@athosflo Looks like the screenshot didn't make the post :-)

Related: It's a lot more helpful to post *raw text*, rather than a screenshot. Raw text can be searched, indexed, and displays on all screens in a zoomable font; a screenshot can be really hard to read, and isn't searchable.

**Comment by freakboy3742** [comment]
Closing as the original issue has apparently been resolved, and the new issue is with notarisation, which is a feature we don't yet support.
